kDriveCore/Data/UploadQueue/Operation/UploadOperation+PHAsset.swift

```diff
@@ -38,6 +38,16 @@ extension UploadOperation {
             return
         }
 
+        // Check if we are not restarting a session for an already imported asset
+        if let existingFileURL = file.pathURL,
+           fileManager.fileExists(atPath: existingFileURL.path) {
+            try transactionWithFile { file in
+                file.uploadingSession?.filePath = existingFileURL.path
```

Comment:
Ok so I tested the code, it works as-is. 
But I think it would be way cleaner to remove `file.uploadingSession.filePath`.
`filePath` It duplicates data, forces us to do an extra DB transaction, and looking at the code it looks more like dead code than anything else.

Reply:
So I did the refactor, feel free to merge or rollback as you wish.